Assets/Scripts/Models/Furniture.cs

```diff
@@ -161,35 +174,51 @@ public float PathfindingModifier
     /// <summary>
     /// Gets or sets the Furniture's pathfinding weight which is multiplied into the Tile's final PathfindingCost.
     /// </summary>
+    /// <value>The pathfinding weight for the tiles the furniture currently occupies.</value>
     public float PathfindingWeight
     {
         get { return pathfindingWeight; }
         set { pathfindingWeight = value; }
     }
-
+    
+    /// <summary>
+    /// Gets the tint used to change the color of the furniture.
+    /// </summary>
+    /// <value>The Color of the furniture.</value>
     public Color Tint { get; private set; }
 
-    // If this furniture gets worked by a person,
-    // where is the correct spot for them to stand,
-    // relative to the bottom-left tile of the sprite.
-    // NOTE: This could even be something outside of the actual
-    // furniture tile itself!  (In fact, this will probably be common).
+    /// <summary>
+    /// Gets the spot where the astronaut will stand when he is using the furniture. This is relative to the bottom
+    /// left tile of the sprite. This can be outside of the actual furniture.
+    /// </summary>
+    /// <value>The spot where the astronaut will stand when he uses the furniture.</value>
     public Vector2 JobSpotOffset { get; private set; }
 
-    // Flag for Lua to check if this is a vertical or horizontal door and display the correct sprite.
+    /// <summary>
+    /// Gets or sets a value indicating whether the door is Vertical or not.
+    /// Should be false if the furniture is not a door.
+    /// </summary>
+    /// <value>Whether the door is Vertical or not.</value>
     public bool VerticalDoor { get; set; }
```

Comment:
Documentation should probably indicate that this field will most probably be removed/move to another class soon.